composer.json

```diff
@@ -18,6 +18,8 @@
         "symfony/options-resolver": "^4.3",
         "symfony/validator": "^4.3",
         "symfony/cache": "^4.3",
+        "symfony/translation": "^4.3",
+        "symfony/translation-contracts": "^1.1.2",
```

Comment:
Why not `^1.1`?

Reply:
> Why not `^1.1`?

Because of composer memory issues. It's better to include latest possible version. Especially now that we don't need to wonder if that breaks someone's project. After 3.0 final we'd need to be more careful, maybe.